internal/db/testing.go

```diff
@@ -229,6 +252,27 @@ func WithTemplate(template string) TestOption {
 	}
 }
 
+// TestLogLevel defines a test log level for the underlying db package (if applicable)
+type TestLogLevel int
+
+const (
+	// See WithTestLogLevel(...) test only option
+	DefaultTestLogLevel TestLogLevel = iota
+	SilentTestLogLevel
+	ErrorTestLogLevel
+	WarnTestLogLevel
+	InfoTestLogLevel
+)
+
+// WithTestLogLevel provides a way to specify a test log level for the
+// underlying database package (if applicable).  To be clear, this has no affect
+// on the events that may be emitted.
```

Comment:
I'm having trouble understanding the last sentence in the context of the `db` package. Can we either remove it or clarify it by adding more details?

Reply:
Happy to remove it... especially if it's just confusing the reader. ty!